src/tdsops.f90

```diff
@@ -56,30 +56,31 @@ module m_tdsops
 
 contains
 
-  function tdsops_init(tds_n, delta, operation, scheme, bc_start, bc_end, &
-                       n_halo, from_to, sym, c_nu, nu0_nu) result(tdsops)
-      !! Constructor function for the tdsops_t class.
-      !!
-      !! 'n', 'delta', 'operation', and 'scheme' are necessary arguments.
-      !! Number of points 'n', distance between two points 'delta', the
-      !! 'operation' the tridiagonal system defines, and the 'scheme' that
-      !! specifies the exact scheme we choose to apply for the operation.
-      !! The remaining arguments are optional.
-      !! 'from_to' is necessary for interpolation and staggared derivative, and
-      !! it can be 'v2p' or 'p2v'.
-      !! If the specific region the instance is operating is not a boundary
-      !! region, then 'bc_start' and 'bc_end' are either 'null' or not defined.
-      !! 'sym' is relevant when the boundary condition is free-slip. If sym is
-      !! .true. then it means the field we operate on is assumed to be an even
-      !! function (symmetric) accross the boundary. If it is .false. it means
-      !! that the field is assumed to be an odd function (anti-symmetric).
-      !! 'c_nu', 'nu0_nu' are relevant when operation is second order
-      !! derivative and scheme is compact6-hyperviscous.
+  function tdsops_init(n_tds, delta, operation, scheme, &
+                       bc_start, bc_end, n_halo, from_to, sym, c_nu, nu0_nu) &
+    result(tdsops)
+    !! Constructor function for the tdsops_t class.
+    !!
+    !! 'n_tds', 'n_rhs', 'delta', 'operation', and 'scheme' are necessary
```

Comment:
`n_rhs` doesn't appear in the arguments list here - and it will be important to document (somewhere) the difference between `n_tds` and `n_rhs`.

Reply:
Fixed, forgot to remove this after changing my approach slightly.